plugins/kotlin/uast/uast-kotlin-base/test/org/jetbrains/uast/test/common/kotlin/UastApiTestBase.kt

```diff
@@ -61,4 +65,21 @@ interface UastApiTestBase : UastPluginSelection {
         )
     }
 
+    fun checkCallbackForSimple(uFilePath: String, uFile: UFile) {
+        val simpleClass = uFile.classes.find { it.name == "Simple" }
+            ?: throw IllegalStateException("Target class not found at ${uFile.asRefNames()}")
+
+        val m = simpleClass.methods.find { it.name == "method" }
+            ?: throw IllegalStateException("Target function not found at ${uFile.asRefNames()}")
+        // type delegated from LC
+        TestCase.assertEquals(PsiType.VOID, m.returnType)
+        // type through the base service
+        val service = ServiceManager.getService(BaseKotlinUastResolveProviderService::class.java)
+        TestCase.assertEquals(PsiType.VOID, service.getType(m.sourcePsi as KtDeclaration, m as UElement))
+
+        val functionCall = m.findElementByText<UElement>("println").uastParent as KotlinUFunctionCallExpression
+        // type through the base service: KotlinUElementWithType#getExpressionType
+        TestCase.assertEquals("PsiType:Unit", functionCall.getExpressionType()?.toString())
```

Comment:
This return type of function call expression _was_ converted to `void` without this change.